### DISCUSSION TITLE
Is there a quick and save method to uninstall KoReader?

### DISCUSSION THREAD

**Discussion by paaguti** [koreader/koreader#11502] [Show and tell]
Hi

I recently got a refurbished Kobo Glo and it has KoReader installed. As I'm quite happy with the built-in reader, I would like to uninstall KoReader on this device for good. (I will try it on an old Cervantes, but that's a longer term project).
Is there a good way to get rid of KoReader and all its stuff?

Thx, /PA

**Comment by Frenzie** [comment]
NickelMenu will uninstall itself if you create a file called uninstall in .adds/nm.

For KOReader you can simply remove 
.adds/koreader

**Comment by paaguti** [comment]
I'll try. Thx for the info

On Thu, 29 Feb 2024 at 07:55, Frans de Jonge ***@***.***>
wrote:

> NickelMenu will uninstall itself if you create a file called uninstall in
> .adds/nm.
>
> For KOReader you can simply remove
> .adds/koreader
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/koreader/koreader/discussions/11502#discussioncomment-8626755>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZKU4EVO2TI5X4T7ISFK4LYV3IF7AVCNFSM6AAAAABD7KP37CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMMRWG42TK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Fragen sind nicht da, um beantwortet zu werden,
Fragen sind da um gestellt zu werden
Georg Kreisler

Headaches with a Juju log:
unit-basic-16: 09:17:36 WARNING juju.worker.uniter.operation we should run
a leader-deposed hook here, but we can't yet

**Comment by BerndHme** [comment]
And also running the KFMon uninstaller as you installed KOReader in the first place I guess?

!!!!!
This is my guess, not certain this is actually correct.
Place in same folder as the install.ps1 script.
Run install.ps1 with powerShell ans select the uninstaller to execute.
!!!!!

Link with thread-answer for question "How to uninstall KOReader?"
post 16
https://www.mobileread.com/forums/showthread.php?t=314220&page=2

Link with thread where the uninstaller is located
https://www.mobileread.com/forums/showthread.php?t=274231
